quickstarts/gigamon/gigamon-hawk/config.yml

```diff
@@ -1,44 +1,50 @@
+# This is automatically created by the system, this field should not be included by the author when creating a new quickstart.
 id: d6b36ecf-bf99-475c-938f-370153db8e70
-slug: gigamon-newrelic
-title: Gigamon
+
+slug: gigamon-hawk
+
+title: Gigamon Hawk Observability
 
 description: |
   Gigamon helps the world’s leading organizations run fast, stay secure and innovate. We provide the industry’s first elastic visibility and analytics fabric, which closes the cloud visibility gap by enabling cloud tools to see the network and network tools to see the cloud. With visibility across their entire hybrid cloud network, organizations can improve customer experience, eliminate security blind spots, and reduce cost and complexity. Gigamon has been awarded over 90 technology patents and enjoys world-class customer satisfaction with more than 4,000 organizations, including over 80 percent of the Fortune 100 and hundreds of government and educational organizations worldwide.
 
   To try this integration in your environment please reach out to tme@gigamon.com. If you are not aware of Gigamon Cloud Suite please reach out to sales@gigamon.com.
 
 summary: |
-
   Gigamon Cloud Suite enables collection of network traffic from every instance in the cloud collecting metadata of over 5,000 network traffic-related attributes for a holistic overview of and a much deeper level of security-related inspection.
 
-# Support level: New Relic | Verified | Community (required)
 level: Verified
 
-# Authors of the quickstart (required)
 authors:
-  - Haider Jarral
+  - Haider Jarral (Gigamon)
 
-# Keywords for filtering / searching criteria in the UI
 keywords:
-  - Gigamon
+  - gigamon
+  - packet capture
+  - network
+  - hawk
   - Gigamon Cloud Suite
   - Gigamon Integration
   - newrelic partner
   - NR1_addData
   - NR1_sys
 
-installPlans:
-  - third-party-gigamon
-
 documentation:
   - name: Gigamon integration guide
     url: >-
       https://www.gigamon.com/content/dam/resource-library/english/deployment-guide/gigamon-new-relic-integration-quick-start-guide.pdf
     description: |
       An overview of the architecture and deployment methodology for Gigamon Hawk integrated with New Relic
 
-icon: logo.png
-website: https://www.gigamon.com
+icon: gigamon-hawk01.png
```

Comment:
```suggestion
icon: logo.png
```

Reply:
The icon file is still named 'logo.png'. You would either have to revert the name here or change the logo name to be the same as what you are calling out here.